boards/nucleo-common/include/board_common.h

```diff
@@ -41,16 +41,18 @@ extern "C" {
  * @{
  */
 #ifdef CPU_MODEL_STM32F302R8
+#define LED0_GPIO           GPIOB
```

Comment:
> what's the difference to here

Ok, I didn't notice that there were already existing macro of this kind.
I agree with the consistence argument, will change that. For the record, I made a similar change in #7085 ([here](https://github.com/RIOT-OS/RIOT/pull/7085/files#diff-8d52b5a7c345c098742d21c5c908f16eR42)) and now that I'm looking at it again, I think it needs to be changed as well...